tests/testlib/s2n_test_certs.c

```diff
@@ -76,3 +76,23 @@ int s2n_read_test_pem_and_len(const char *pem_path, uint8_t *pem_out, uint32_t *
     return 0;
 }
 
+S2N_RESULT s2n_test_cert_chain_from_pem(struct s2n_connection *conn, const char *pem_path,
+        struct s2n_stuffer *cert_chain_stuffer) {
+    RESULT_ENSURE_REF(cert_chain_stuffer);
+    RESULT_GUARD_POSIX(s2n_stuffer_growable_alloc(cert_chain_stuffer, 4096));
+
+    uint8_t cert_chain_pem[S2N_MAX_TEST_PEM_SIZE] = { 0 };
+    uint32_t cert_chain_pem_len = 0;
+    RESULT_GUARD_POSIX(s2n_read_test_pem_and_len(pem_path, cert_chain_pem, &cert_chain_pem_len, S2N_MAX_TEST_PEM_SIZE));
+
+    DEFER_CLEANUP(struct s2n_cert_chain_and_key *chain_and_key = s2n_cert_chain_and_key_new(),
+            s2n_cert_chain_and_key_ptr_free);
+    RESULT_GUARD_POSIX(s2n_cert_chain_and_key_load_public_pem_bytes(chain_and_key, cert_chain_pem, cert_chain_pem_len));
+
```

Comment:
Ah, you can't use s2n_test_cert_chain_and_key_new because you don't have a private key. That also explains why this method originally didn't do this; not needing a private key is a pretty recent feature. Makes sense!